pkgs/applications/misc/monitor/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, vala
+, pkgconfig
+, pantheon
+, python3
+, gettext
+, glib
+, gtk3
+, bamf
+, libwnck3
+, libgee
+, libgtop
+, libxml2
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "monitor";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "stsdc";
+    repo = "monitor";
+    rev = version;
+    sha256 = "17z1m193s7qygavfwd8qsw97blxbfmq9gnsymdjlc1ddk8hldw0z";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    meson
+    ninja
+    vala
+    pkgconfig
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    bamf
+    glib
+    gtk3
+    pantheon.granite
+    pantheon.wingpanel
+    libgee
+    libgtop
+    libwnck3
+    libxml2
+  ];
+
+  PKG_CONFIG_WINGPANEL_2_0_INDICATORSDIR = "${placeholder "out"}/lib/wingpanel";
```

Comment:
```suggestion
```

I've created this PR to eliminate this issue https://github.com/stsdc/monitor/pull/141. Can you fetch the patches?

Reply:
Building with your patch did eliminate the issue, but I still can't see the the indicator in Wingpanel :sweat_smile: